src/commands/dev/lib/yarn-installer.ts

```diff
@@ -0,0 +1,71 @@
+import execa from 'execa';
+import { tmpdir } from 'os';
+import {
+  pathExists,
+  mkdirp,
+  createWriteStream,
+  statSync,
+  chmodSync
+} from 'fs-extra';
+import pipe from 'promisepipe';
+import { join } from 'path';
+import fetch from 'node-fetch';
+import tar from 'tar-fs';
+import { setScriptGlobals } from '@now/build-utils';
+import { Output } from '../../../util/output/create-output';
+
+async function isFoundInPath(): Promise<boolean> {
+  const [npm, yarn] = await Promise.all([
+    execa('npm', ['--version']),
```

Comment:
Won't these throw an error if npm/yarn is not installed?

Reply:
@styfle Yea, let's run `which-promise` here instead, please

Reply:
Fixed!